changedetectionio/tests/test_xpath_selector.py

```diff
@@ -277,3 +277,45 @@ def test_check_with_prefix_include_filters(client, live_server):
     assert b"Some text that will change" not in res.data #not in selector
 
     client.get(url_for("form_delete", uuid="all"), follow_redirects=True)
+
+def test_xpath_20(client, live_server):
+    res = client.get(url_for("form_delete", uuid="all"), follow_redirects=True)
+    assert b'Deleted' in res.data
+
+    # Give the endpoint time to spin up
+    time.sleep(1)
+
+    set_original_response()
+
+    # Add our URL to the import page
+    test_url = url_for('test_endpoint', _external=True)
+    res = client.post(
+        url_for("import_page"),
+        data={"urls": test_url},
+        follow_redirects=True
+    )
+    assert b"1 Imported" in res.data
+    time.sleep(3)
+
+    res = client.post(
+        url_for("edit_page", uuid="first"),
+        data={"include_filters": "//*[contains(@class, 'sametext')]|//*[contains(@class, 'changetext')]",
```

Comment:
Parsing with the union operator.  The operator is supported since 1.0.
```
//*[contains(@class, 'sametext')]
//*[contains(@class, 'changetext')]
```
will parse like 
```
<div class="sametext">Some text thats the same</div>
<div class="sametext">Some text thats the same</div>
<div class="changetext">Some new text</div>
<div class="changetext">Some new text</div>

```

But the operator will parse like this
```
<div class="sametext">Some text thats the same</div>
<div class="changetext">Some new text</div>
<div class="sametext">Some text thats the same</div>
<div class="changetext">Some new text</div>
```

Reply:
Does this mean that it affects existing xpaths that people have saved?

Reply:
No. It won't. It is just an explanation. I didn't explain here enough. I'm sorry. 

Let's say `modified.html` contains the content below.
```
$cat modified.html
<html>
<body>
<p>Which is across multiple lines</p>
<div class="sametext">Some text thats the same</div>
<div class="changetext">Some new text</div>
<div class="sametext">Some text thats the same</div>
<div class="changetext">Some new text</div>
</body>
</html>
```

`basex` is cli xpath parser.
If the user uses the XPath union syntax, the result is below.
```
$basex -i modified.html "//*[contains(@class, 'sametext')]|//*[matches(@class, 'changetext')]" 
<div class="sametext">Some text thats the same</div>
<div class="changetext">Some new text</div>
<div class="sametext">Some text thats the same</div>
<div class="changetext">Some new text</div>
```